lib/zipkin-tracer/rack/zipkin-tracer.rb

```diff
@@ -59,6 +59,16 @@ def call(env)
     end
 
     private
+
+    # If the request is not valid for this service, we do not what to trace it.
+    def valid_request?(env)
```

Comment:
Can the naming be improved? `valid_` is too broad IMHO. `processable_` or something?